.github/workflows/visual-tests.yml

```diff
@@ -52,7 +56,8 @@ jobs:
           workers: 1
           extra-options: >-
             ${{ inputs.update-snapshots && ' --update-snapshots' || '' }}
-            ${{ inputs.filter && format(' --grep="{0}"', inputs.filter) || '' }}
+            ${{ inputs.include && format(' --test-match="**/{0}/**/*.spec.ts"', inputs.include) || '' }}
+            ${{ inputs.exclude && format(' --test-ignore="**/{0}/**/*.spec.ts"', inputs.exclude) || '' }}
```

Comment:
Since these flags don't exist, as we already aligned on via Slack, you could instead pass the paths to include/exclude as arguments [^1]:

| Option | Description |
|---|---|
| Non-option arguments | Each argument is treated as a regular expression matched against the full test file path. Only tests from files matching the pattern will be executed. Special symbols like $ or * should be escaped with \. In many shells/terminals you may need to quote the arguments. |

[^1]: https://playwright.dev/docs/test-cli#all-options